docs/02.1-bundlers.mdx

```diff
@@ -130,3 +130,174 @@ Configuring webpack is not for the faint of heart, we recommend thoroughly readi
 [Our template](https://github.com/excaliburjs/template-ts-webpack) is a good starting point.
 
 </Note>
+
+### Deno
+
+[Deno](https://deno.land/) is a runtime for JavaScript and TypeScript and much more than a bundler, but we put this section here since you'll be doing a lot of bundling :).
+
+For Excalibur to work in a Deno envirionment, use a Content Delivery Network for making Node.js packages compatible with Deno: two popular ones are [esm.sh](https://esm.sh/) and [skypack.dev](https://www.skypack.dev/).
+
+To keep it simple, we use esm.sh in the following examples, but feel free to make your own choice - at the time of writing the procedure for esm.sh and skypack.dev is nearly identical (for a difference, see the section [Types and IntelliSense](#types-and-intellisense)).
+
+#### Importing within an HTML file
+
+Sometimes, you just want to start hacking. You can do so by using Deno to get a bundled version of Excalibur and directly applying the module to your HTML.
+
+Either [esm.sh](https://esm.sh/excalibur) or [skypack.dev](https://www.skypack.dev/view/excalibur) should generate a bundle that works:
+
+```bash
+deno bundle https://esm.sh/excalibur excalibur.bundle.js
+```
+
+<Note>
+
+Learn more about the `deno bundle` command in the [official docs](https://deno.land/manual/tools/bundler#bundling).
+
+</Note>
+
+Using as a native module works in Chrome without any extras:
+
+```html
+<!-- index.html -->
+
+<script type="module">
+  import ex from './excalibur.bundle.js'
+
+  const game = new ex.Engine()
+  game.start()
+</script>
+```
+
+The drawback to this method is that it doesn't give you type annotations in your IDE, like [the following method](#importing-into-a-javascript-or-typescript-file).
+
+<Note>
+
+Learn more about the `type="module"` attribute [on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).
+
+</Note>
+
+#### Importing into a JavaScript or TypeScript File
+
+<Note>
+
+We have tested the following examples with version 0.26.0-alpha.264 of the excalibur package. We recommend you run at least that version for this to work. You can get [the latest excalibur version on npm](https://www.npmjs.com/package/excalibur).
```

Comment:
We'll cut a point release soon, I'll make a note so I don't forget to update this :+1: